itest/lnd_open_channel_test.go

```diff
@@ -877,6 +877,114 @@ func testFundingExpiryBlocksOnPending(ht *lntest.HarnessTest) {
 	ht.MineBlocksAndAssertNumTxes(1, 1)
 }
 
+// testConfirmationsUntilActiveOnPending verifies that as a channel status
```

Comment:
Nit: testConfirmationsUntilActiveOnPending => testPendingChannelConfirmationUntilActive verifies the value for the rpc filed `ConfirmationUntilActive` decreases as expected as soon as blocks are confirmed.